.github/workflows/run-fluent-updates.yml

```diff
@@ -0,0 +1,17 @@
+name: "Run Fluent Updates"
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: "1,16,31,46 * * * *" # Every 15 mins, starting at 1min past the hour
```

Comment:
It's currently run on a schedule, but it'd be better if it was run whenever changes land in the [l10n team fluent repo](https://github.com/mozilla-l10n/www-l10n).

Reply:
I'll add a webhook to join the dots.

We'll also have to disable the DMS ping (or give it a really big window), because we won't be running an update as regularly as every 15mins

Reply:
maybe we can have it run once or twice per day, just to be sure we didn't miss a ping, and have DMS set for that?

Reply:
Yeah that seems nice - scheduled run every 12hrs and the DMS waiting threshold is increased from 15min to ~12.5 hours 👍